Assets/_Scripts/Systems/PlayerSystem.cs

```diff
@@ -200,7 +200,7 @@ public void RemovePlayer(PlayerEntity player)
 			
 			player.TriggerDespawn();
 			playersEntity.Remove(player);
-			playersJoined.Remove(player.Object.InputAuthority);
+			playersJoined.Remove(player.Object.InputAuthority); //TODO fix null reference
```

Comment:
J'imagine qu'il n'y a plus de null reference?

Reply:
SQUASH IT